src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -4,5 +4,20 @@
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
+    public static final int COUNT_FIGURES = 6;
```

Comment:
The constant `COUNT_FIGURES` is defined correctly, but ensure that its value aligns with the intended number of figures to be created. If the number of figures changes, remember to update this constant accordingly.